### DISCUSSION TITLE
What is the status of the TimePicker component

### DISCUSSION THREAD

**Discussion by r25s** [chakra-ui/ark#2709] [General]
Hello good folks,

I noticed that Ark ships with a Time Picker component, and the README says that Time Picker is available. However there is no documentation for it on the website. I'm sure there is a reason for this, just wondering what it is. And is there any preliminary documentation available?

Thanks!

**Comment by segunadebayo** [comment]
Not ready for production use. It's quite buggy and needs to be covered with tests for confidence

**Reply by r25s** [reply]
Thanks for your reply!

**Reply by Thesmit113** [reply]
@segunadebayo
Hey! Just checking in—any updates on the time picker? Excited for its release and looking forward to using it soon!

Thanks! 😊
